uui-core/services/ApiContext.ts

```diff
@@ -28,6 +27,7 @@ export interface FileUploadResponse {
     path?: string;
     type?: BlockTypes;
     extension?: string;
+    uploadError?: { isError: boolean, message?: string };
```

Comment:
lets move { isError: boolean, message?: string } on the first level of interface